payu/subcommands/args.py

```diff
@@ -191,6 +191,17 @@
     }
 }
 
+# Clone branch
+clone_start_point = {
+    'flags': ('--start-point', '-s'),
+    'parameters': {
+        'action':   'store',
+        'dest': 'start_point',
+        'default':  None,
+        'help': 'New branch will start from this commit or tag'
+    }
+}
```

Comment:
Is there a reason not to re-use the existing `start_point` option for this case?

Reply:
Yes, for the checkout command `start_point` is an optional positional argument (to be more similar to `git checkout -b NEW_BRANCH  START_POINT`)